fastlane/lib/fastlane/actions/docs/frame_screenshots.md

```diff
@@ -68,18 +81,70 @@ To download the latest frames
 
     fastlane frameit download_frames
 
-When using _frameit_ without titles on top, the screenshots will have the full resolution, which means they can't be uploaded to the App Store directly. They are supposed to be used for websites, print media and emails. Check out the section below to use the screenshots for the App Store.
+When using _frameit_ without titles on top, the screenshots will have the full resolution, which means they can't be uploaded to the App Store directly. They are supposed to be used for websites, print media and emails. Check out the section below to use the screenshots for the App Store. TODO Option to also scale those?
+
+## Advanced Usage (optional)
 
-# Titles and Background (optional)
+### Text and Background
 
-With _frameit_ 2.0 you are now able to add a custom background, title and text colors to your screenshots.
+With _frameit_ it's possible to add a custom background and text below or above the framed screenshots in fonts and colors you define.
 
 A working example can be found in the [fastlane examples](https://github.com/fastlane/examples/tree/master/MindNode/screenshots) project.
 
 #### `Framefile.json`
 
-Use it to define the general information:
+The Framefile allows to define general and screenshot specific information.
+It has the following JSON structure:
 
+```json
+{
+  "device_frame_version": "latest",
+  "default": {
+    ...
+  },
+  "data": [
+     ...
+  ]
+}
+```
+
+TODO list parameters first, then show example
+
+### General parameters
+
+The general parameters are defined in the `default` key and can be:
+
+| Key | Description | Default value |
```

Comment:
missing: `offset` for Mac https://docs.fastlane.tools/actions/frameit/#example

Reply:
Shall we make an extra table with the Mac specific parameters?

Reply:
Sounds like a good iea, yeah.